File: python/raindrops/raindrops.py
MODULO_MAPPER = {
    3: 'Pling',
    5: 'Plang',
    7: 'Plong'
}


def convert(number):
    answer = ''

    for k, v in MODULO_MAPPER.items():
        if number % k == 0:
            answer += v
    if not answer:
        answer = str(number)

    return answer

File: javascript/raindrops/raindrops.js
//
// This is only a SKELETON file for the 'Raindrops' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const convert = (number) => {
  let answer = '';

  if (number % 3 == 0) {
    answer += 'Pling';
  }
  if (number % 5 == 0) {
    answer += 'Plang';
  }
  if (number % 7 == 0) {
    answer += 'Plong';
  }

  return (answer || number.toString());
};
